docs/pages/custom-builds/functions.mdx

```diff
@@ -87,7 +83,13 @@ This command triggers the `build` script placed in the **package.json** file of
 
 The `build` script generates **build/index.js**. This file must be uploaded to EAS Build as a part of your project archive, so that the builder can run your function. Ensure that the file is not excluded by a **.gitignore** file or **.easignore** file.
 
-### Connect the function to the custom build config
+</Step>
+
+<Step label="3">
+
+## Expose the function to the custom build config
+
+> **Note:** The following example assumes that you have already set up a custom build workflow and configured it in your **eas.json**. If you haven't, see our [“Get started with custom builds” guide](/custom-builds/get-started/#create-a-workflow) first.
```

Comment:
```suggestion
> **Note**: The following example assumes that you have already set up a custom build workflow and configured it in your **eas.json**. If not, see [Get started with custom builds](/custom-builds/get-started/#create-a-workflow) before proceeding.
```

- No quotes in when referring internal link. The link color highlight is enough to grab attention of a user if they don't skim the note.
- Also, we don't have to explicitly call out the type of the document here.

Reply:
thanks!